.circleci/config.yml

```diff
@@ -199,3 +204,18 @@ jobs:
     environment:
       HELM_VERSION: v2.9.1
       GKE_BRANCH: 1.10
+  sync_chart:
+    docker:
+      - image: circleci/golang:1.9
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "cc:ed:2d:1f:74:3b:32:c3:99:62:d7:7d:81:af:01:e0"
+      - run: |
+          if ls ~/.ssh/id_rsa_* 1> /dev/null 2>&1; then
+            # Change order to use configured ssh_key first
+            ssh-add -D
+            ssh-add ~/.ssh/id_rsa_* ~/.ssh/id_rsa
+          fi
+          ./script/chart_sync.sh kubernetes-bitnami kubernetes@bitnami.com
```

Comment:
just curious, why do we use this bot instead of bitnami-bot? It might be easier to use bitnami-bot since it's already setup to push to bitnami/charts and will look more consistent, but I don't think it matters too much. On the other hand, using this bot will prevent this from breaking if there are changes to bitnami-bot somewhere else.

Reply:
It's a busy bot 🤖 🤣